.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!--
+  Thanks for opening an issue! A few things to keep in mind:
+  
+  - The issue tracker is only for bugs and feature requests.
+  - Before reporting a bug, please make sure you have the latest version of Webviz and try to do a clean install of the project.
```

Comment:
Also add something along the lines of: , ensure the bug has not been reported before
Or is that our responsibility?

Reply:
I think it is fair to require a quick search before creating a new issue. Maybe something along the lines of:

> Before opening a new issue, please search for similar issue among existing and closed issues.